vm/vm.go

```diff
@@ -40,3 +41,16 @@ func NewHandler(endpoints map[Type]string) *Handler {
 		instanceMgr: server.NewMgr(),
 	}
 }
+
+var DefaultHandler *Handler
+
+func init() {
+	var endpoints = make(map[Type]string)
+	for key, typ := range vmEndpointConfigEnvKeyMap {
+		if ep := viper.GetString(key); ep != "" {
+			endpoints[typ] = ep
+		}
+	}
+
+	DefaultHandler = NewHandler(endpoints)
+}
```

Comment:
why we need it be initialized in `init`?

Reply:
I tried to initialize a default vm handler, if the vm endpoint can be read from env. And then the `vm.DefaultHandler` will be used to initialize the `msg.DefaultHandler`.

Reply:
if some module don't need the default handler, they can create a hander by `NewHander`

Reply:
i  think default if unnecessary， just NewHandler is more clear